cogs/err.py

```diff
@@ -47,11 +47,11 @@ def __init__(self, bot):
             summ = (rc >> 21) & 0x3F
             level = (rc >> 27) & 0x1F
             embed = discord.Embed(title=f"0x{rc:X}")
-            embed.add_field(name="Module", value=dds_modules[mod])
+            embed.add_field(name="Module", value=dds_modules.get(mod, mod))
```

Comment:
Have you confirmed that the implicit int -> str conversion doesn't cause any issues?

Reply:
Just confirmed that it works.